src/reducers/MapReducer.js

```diff
@@ -82,39 +95,59 @@ const updateFilterKeys = ({filterCriteriaKeys}, filterKey) => {
     return updatedFilterKeyList;
 };
 
-const filterSites = ({sites}, updatedFilterKeys) => {
-    if (updatedFilterKeys.accessibility.length === 0 && updatedFilterKeys.facilities.length === 0 && updatedFilterKeys.price.length === 0) {
+const filterSites = ({sites, filterResultsScrutinyLoose}, updatedFilterKeys) => {
+    if (updatedFilterKeys.accessibility.length === 0 && updatedFilterKeys.facilities.length === 0 && updatedFilterKeys.features.length === 0 && updatedFilterKeys.price.length === 0) {
         return sites;
     }
 
     const filteredSites = _.filter(sites, site => {
-        const {accessibility, price, facilities} = site;
+        const {accessibility, price, facilities, features} = site;
         let accessibilityMatch = false;
         let facilitiesMatch = false;
+        let featuresMatch = false;
         let priceMatch = false;
 
         if (updatedFilterKeys.accessibility.length === 0) {
             accessibilityMatch = true;
         } else {
-            const accessibilityFullTextArray = _.map(updatedFilterKeys.accessibility, filterKey => accessibility_options[filterKey]);
-            accessibilityMatch = _.includes(accessibilityFullTextArray, accessibility);
+            accessibilityMatch = _.includes(updatedFilterKeys.accessibility, accessibility);
         }
 
         if (updatedFilterKeys.facilities.length === 0) {
             facilitiesMatch = true;
         } else {
-            const facilitiesFullTextArray = _.map(updatedFilterKeys.facilities, filterKey => facilities_options[filterKey]);
-            facilitiesMatch = _.includes(facilitiesFullTextArray, facilities);
+            if (filterResultsScrutinyLoose.facilities) {
+                facilitiesMatch = _.some(updatedFilterKeys.facilities, facilityFromFilterList => {
+                    return _.includes(facilities, facilityFromFilterList);
+                });
+            } else {
+                facilitiesMatch = _.every(updatedFilterKeys.facilities, facilityFromFilterList => {
+                    return _.includes(facilities, facilityFromFilterList);
+                });
+            }
+        }
+
+        if (updatedFilterKeys.features.length === 0) {
+            featuresMatch = true;
+        } else {
+            if (filterResultsScrutinyLoose.features) {
+                featuresMatch = _.some(updatedFilterKeys.features, featureFromFilterList => {
+                    return _.includes(features, featureFromFilterList);
+                });
+            } else {
+                featuresMatch = _.every(updatedFilterKeys.features, featureFromFilterList => {
+                    return _.includes(features, featureFromFilterList);
+                });
```

Comment:
If `filterResultsScrootinyLoose` is `false` (which means we *should* have high scrutiny) we need to check that every one of the filter criteria are matched in the given site. If not, and it is loose, we can just check to see if that site contains *any* of the criteria in the filter list.